docs/checks/docs_link_checker.py

```diff
@@ -77,7 +77,7 @@ def __init__(
         self._absolute_link_pattern = re.compile(absolute_link_regex)
 
         # docroot links start without a . or a slash
-        docroot_link_regex = r"^(?P<path>\w[\.\w\/-]+\.md)(?:#\S+)?$"
+        docroot_link_regex = r"^(?P<path>\/\w[\.\w\/-]+\.md)(?:#\S+)?$"
```

Comment:
regex was not handling the first `/`

Reply:
but docroot links start _without_ a . or / per the comment